table/block_based_table_builder.cc

```diff
@@ -581,10 +617,20 @@ void BlockBasedTableBuilder::WriteBlock(const Slice& raw_block_contents,
     }
     assert(compression_dict != nullptr);
     CompressionInfo compression_info(r->compression_opts, r->compression_ctx,
-                                     *compression_dict, r->compression_type);
-    block_contents =
-        CompressBlock(raw_block_contents, compression_info, &type,
-                      r->table_options.format_version, &r->compressed_output);
+                                     *compression_dict, type, sample_for_compression);
+
+    std::string sampled_output_fast;
+    std::string sampled_output_slow;
+    block_contents = CompressBlock(raw_block_contents, compression_info,
+                                   &type, r->table_options.format_version,
+                                   is_data_block, /* do_sample */
```

Comment:
Nit: `is_data_block /* do_sample */,`. The comma should be between `is_data_block` and the comment.

Reply:
Fixed.